src/System.Globalization/tests/DateTimeStyles/DateTimeStylesTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Globalization.Tests
+{
+    public class DateTimeStylesTests
+    {
+        [Theory]
+        [InlineData(DateTimeStyles.AllowInnerWhite, 0x00000004)]
+        [InlineData(DateTimeStyles.AssumeLocal, 0x00000020)]
+        [InlineData(DateTimeStyles.AllowWhiteSpaces, 0x00000001 | 0x00000002 | 0x00000004)]
+        [InlineData(DateTimeStyles.AllowTrailingWhite, 0x00000002)]
+        [InlineData(DateTimeStyles.AdjustToUniversal, 0x00000010)]
+        [InlineData(DateTimeStyles.AllowLeadingWhite, 0x00000001)]
+        [InlineData(DateTimeStyles.None, 0x00000000)]
+        [InlineData(DateTimeStyles.AssumeUniversal, 0x00000040)]
+        [InlineData(DateTimeStyles.NoCurrentDateDefault, 0x00000008)]
+        [InlineData(DateTimeStyles.RoundtripKind, 0x00000080)]
+        public void Cases(DateTimeStyles style, ulong expected)
```

Comment:
Nit: This is a strange name for the test.  I'd expect something more descriptive like "EnumNames_MatchExpectedValues", or something like that.